### PR TITLE
Switch to master branch

### PR DESCRIPTION
**Pull request by davetcoleman**
- Document what the master branch is
- Add Travis badge
- Update rosinstall

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,21 @@ Currently we support ROS Indigo, Kinetic, and Melodic.
 - [Documentation](http://moveit.ros.org/documentation/)
 - [Get Involved](http://moveit.ros.org/documentation/contributing/)
 
+## Branches Policy
+
+We develop all new 1.0 features on ``master``. The ``*-devel`` branches are actually treated
+like release branches, corresponding to stable version of MoveIt for
```
Comment (130s) [inline README.md]: Changing `like release branches` -> `like master branches` might make more sense to broader audience.

Assuming more people are familiar with common development flow, such as [git flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) or its variants, where new change is made on "development" branch and the only released artifact goes into "master".

Comment (gavanderhoorn) [reply README.md]: I might actually agree with @130s here, and have suggested something similar in the past. Using `-devel` branches for releases (and stating that they are "stable") is rather counter-intuitive and confusing.

For other projects I've been using `-devel` branches for actual development and a corresponding release branch (that syncs with that branch every now and then) for actual releases. With the setup here in this repository, that could mean creating a `melodic` release branch that gets synced from `melodic-devel` whenever a release is prepared.

Comment (davetcoleman) [reply README.md]: I attempted to reword it just now. Yes, development can happen in those branches but I think that should be discouraged. I also think renaming these branches would be more hassle than its worth. We've long considered ``hydro-devel`` ``indigo-devel`` etc to be stable but we still call them devel. I know plenty of projects which depend on this branches for their rosinstall build configuration, and I don't see the value in breaking that.

Comment (gavanderhoorn) [reply README.md]: They don't need to be renamed.

Release branches are typically created next to development branches.

Comment (davetcoleman) [reply README.md]: I don't see the need for additional release branches for melodic and before, but going forward - now that we have a master branch we probably do need a release branch. For example, when we released moveit 1.1, we need a branch for that. Maybe we call it ``noetic-1.1``?

**Pull request merged**